Source/WebInspectorUI/UserInterface/Views/RecordingActionTreeElement.js

```diff
@@ -101,6 +105,7 @@ WI.RecordingActionTreeElement = class RecordingActionTreeElement extends WI.Gene
             case WI.Recording.Swizzle.WebGLVertexArrayObject:
                 parameterElement.classList.add("swizzled");
                 parameterElement.textContent = WI.Recording.displayNameForSwizzleType(swizzleType);
+                parameterElement.textContent += ": " + parameter;
```

Comment:
If someone wants to style this differently, I'd welcome it. I'm not a UI designer.

Reply:
I like to draw something like the little colored letters in this image. 
![image](https://webkit.org/wp-content/uploads/Retained-Size.png)